### PR TITLE
feat: add ftchinese parser

### PR DESCRIPTION
**Pull request by HenryQW**
It seems the parser can't handle Chinese characters for content, but works for title, author and dek.

Preview is showing a bunch of unicode rather than the actual Chinese characters.

```json
{
  "title": "欧盟否决西门子-阿尔斯通铁路业务合并案",
  "content": "<div><div id=\"story-body-container\"><p>&#x5FB7;&#x56FD;&#x548C;&#x6CD5;&#x56FD;&#x94C1;&#x8DEF;&#x8BBE;&#x5907;&#x5236;&#x9020;&#x5546;&#x897F;&#x95E8;&#x5B50;(Siemens)&#x4E0E;&#x963F;&#x5C14;&#x65AF;&#x901A;(Alstom)&#x4E4B;&#x95F4;&#x7684;&#x5408;&#x5E76;&#x5DF2;&#x88AB;&#x6B27;&#x76DF;&#x7ADE;&#x4E89;&#x6267;&#x6CD5;&#x673A;&#x6784;&#x5426;&#x51B3;&#xFF0C;&#x76D1;&#x7BA1;&#x8005;&#x4E0D;&#x63A5;&#x53D7;&#x8FD9;&#x6837;&#x4E00;&#x4E2A;&#x89C2;&#x70B9;&#xFF0C;&#x5373;&#x8FD9;&#x4E00;&#x8054;&#x59FB;&#x5BF9;&#x4E8E;&#x6B27;&#x6D32;&#x4E0E;&#x7531;&#x56FD;&#x5BB6;&#x652F;&#x6301;&#x7684;&#x4E2D;&#x56FD;&#x7ADE;&#x4E89;&#x5BF9;&#x624B;&#x7ADE;&#x4E89;&#x662F;&#x5FC5;&#x8981;&#x7684;&#x3002;</p><p>&#x6B27;&#x76DF;&#x7ADE;&#x4E89;&#x4E8B;&#x52A1;&#x4E13;&#x5458;&#x739B;&#x683C;&#x4E3D;&#x7279;&#x2022;&#x7EF4;&#x65AF;&#x7279;&#x683C;(Margrethe Vestager)&#x5468;&#x4E09;&#x5BA3;&#x5E03;&#x4E86;&#x8FD9;&#x9879;&#x5426;&#x51B3;&#x3002;&#x6B64;&#x524D;&#xFF0C;&#x5176;&#x4ED6;&#x6B27;&#x76DF;&#x4E13;&#x5458;&#x4EE5;&#x53CA;&#x5404;&#x56FD;&#x76D1;&#x7BA1;&#x673A;&#x6784;&#x652F;&#x6301;&#x5979;&#x7684;&#x7ACB;&#x573A;&#xFF0C;&#x5373;&#x8FD9;&#x7B14;&#x4EA4;&#x6613;&#x53EF;&#x80FD;&#x5BFC;&#x81F4;&#x4EF7;&#x683C;&#x4E0A;&#x6DA8;&#x5E76;&#x9650;&#x5236;&#x9009;&#x62E9;&#xFF0C;&#x6700;&#x7EC8;&#x53EF;&#x80FD;&#x6709;&#x635F;&#x6B27;&#x6D32;&#x4E58;&#x5BA2;&#x7684;&#x5229;&#x76CA;&#x3002;</p><p>&#x201C;&#x5982;&#x679C;&#x6CA1;&#x6709;&#x8DB3;&#x591F;&#x7684;&#x8865;&#x6551;&#x63AA;&#x65BD;&#xFF0C;&#x8FD9;&#x4E00;&#x5408;&#x5E76;&#x5C06;&#x5BFC;&#x81F4;&#x4FDD;&#x8BC1;&#x4E58;&#x5BA2;&#x5B89;&#x5168;&#x4EE5;&#x53CA;&#x7528;&#x4E8E;&#x4E0B;&#x4E00;&#x4EE3;&#x9AD8;&#x901F;&#x5217;&#x8F66;&#x7684;&#x4FE1;&#x53F7;&#x7CFB;&#x7EDF;&#x4EF7;&#x683C;&#x4E0A;&#x6DA8;&#xFF0C;&#x201D;&#x7EF4;&#x65AF;&#x7279;&#x683C;&#x8868;&#x793A;&#x3002;&#x5979;&#x8865;&#x5145;&#x79F0;&#xFF1A;&#x201C;&#x4E24;&#x5BB6;&#x516C;&#x53F8;&#x4E0D;&#x613F;&#x610F;&#x5E94;&#x5BF9;&#x6211;&#x4EEC;&#x4E25;&#x91CD;&#x7684;&#x7ADE;&#x4E89;&#x62C5;&#x5FE7;&#x3002;&#x201D;</p><p>&#x8FD9;&#x4E00;&#x5426;&#x51B3;&#x5C06;&#x52A0;&#x5267;&#x4E00;&#x573A;&#x957F;&#x671F;&#x8FA9;&#x8BBA;&#xFF0C;&#x7126;&#x70B9;&#x95EE;&#x9898;&#x662F;&#x6B27;&#x76DF;&#x4E25;&#x683C;&#x7684;&#x53CD;&#x5784;&#x65AD;&#x89C4;&#x5219;&#x662F;&#x5426;&#x9700;&#x8981;&#x66F4;&#x65B0;&#xFF0C;&#x4EE5;&#x4FBF;&#x8BA9;&#x6B27;&#x76DF;&#x4F01;&#x4E1A;&#x4E0E;&#x7531;&#x56FD;&#x5BB6;&#x652F;&#x6301;&#x7684;&#x4E2D;&#x56FD;&#x4F01;&#x4E1A;&#x5C55;&#x5F00;&#x7ADE;&#x4E89;&#x3002;&#x5728;&#x5404;&#x81EA;&#x653F;&#x5E9C;&#x7684;&#x652F;&#x6301;&#x4E0B;&#xFF0C;&#x897F;&#x95E8;&#x5B50;&#x548C;&#x963F;&#x5C14;&#x65AF;&#x901A;&#x8868;&#x793A;&#xFF0C;&#x5408;&#x5E76;&#x5176;&#x94C1;&#x8DEF;&#x4E1A;&#x52A1;&#x5BF9;&#x4E8E;&#x62B5;&#x5FA1;&#x6765;&#x81EA;&#x4E2D;&#x56FD;&#x4E2D;&#x8F66;(CRRC)&#x7684;&#x7ADE;&#x4E89;&#x662F;&#x5FC5;&#x8981;&#x7684;&#xFF1B;&#x540E;&#x8005;&#x662F;&#x4E16;&#x754C;&#x6700;&#x5927;&#x7684;&#x94C1;&#x8DEF;&#x8F66;&#x8F86;&#x5236;&#x9020;&#x5546;&#x3002;</p><p>&#x7136;&#x800C;&#xFF0C;&#x7ADE;&#x4E89;&#x5B98;&#x5458;&#x4EEC;&#x5F97;&#x51FA;&#x7ED3;&#x8BBA;&#x8BA4;&#x4E3A;&#xFF0C;&#x4E2D;&#x56FD;&#x4F01;&#x4E1A;&#x4E0D;&#x592A;&#x53EF;&#x80FD;&#x5F88;&#x5FEB;&#x5728;&#x6B27;&#x6D32;&#x9500;&#x552E;&#x706B;&#x8F66;&#xFF0C;&#x8FD9;&#x610F;&#x5473;&#x7740;&#x6B27;&#x76DF;&#x4E25;&#x683C;&#x7684;&#x53CD;&#x5784;&#x65AD;&#x89C4;&#x5219;&#x8981;&#x6C42;&#x8FD9;&#x4E24;&#x5BB6;&#x516C;&#x53F8;&#x51FA;&#x552E;&#x53EF;&#x89C2;&#x7684;&#x706B;&#x8F66;&#x548C;&#x4FE1;&#x53F7;&#x8D44;&#x4EA7;&#x3002;</p><p>&#x6CD5;&#x56FD;&#x8D22;&#x653F;&#x90E8;&#x957F;&#x5E03;&#x9C81;&#x8BFA;&#x2022;&#x52D2;&#x6885;&#x5C14;(Bruno Le Maire)&#x5F62;&#x5BB9;&#x7EF4;&#x65AF;&#x7279;&#x683C;&#x7684;&#x51B3;&#x5B9A;&#x201C;&#x662F;&#x4E00;&#x4E2A;&#x653F;&#x6CBB;&#x9519;&#x8BEF;&#xFF1A;&#x6B27;&#x76DF;&#x59D4;&#x5458;&#x4F1A;&#x7684;&#x804C;&#x8D23;&#x662F;&#x634D;&#x536B;&#x6B27;&#x6D32;&#x7684;&#x7ECF;&#x6D4E;&#x5229;&#x76CA;&#x2026;&#x2026;&#x62D2;&#x7EDD;&#x963F;&#x5C14;&#x65AF;&#x901A;&#x4E0E;&#x897F;&#x95E8;&#x5B50;&#x5408;&#x5E76;&#x5C06;&#x6709;&#x5229;&#x4E8E;&#x4E2D;&#x56FD;&#x7684;&#x7ECF;&#x6D4E;&#x548C;&#x4EA7;&#x4E1A;&#x5229;&#x76CA;&#x201D;&#x3002;&#x8FD9;&#x4E9B;&#x60C5;&#x7EEA;&#x5F97;&#x5230;&#x5FB7;&#x56FD;&#x7ECF;&#x6D4E;&#x90E8;&#x957F;&#x5F7C;&#x5F97;&#x2022;&#x963F;&#x5C14;&#x7279;&#x8FC8;&#x5C14;(Peter Altmaier)&#x7684;&#x547C;&#x5E94;&#xFF0C;&#x4ED6;&#x547C;&#x5401;&#x4FEE;&#x6539;&#x6B27;&#x76DF;&#x7ADE;&#x4E89;&#x89C4;&#x5219;&#xFF0C;&#x4F7F;&#x5927;&#x578B;&#x5E76;&#x8D2D;&#x4EA4;&#x6613;&#x53D8;&#x5F97;&#x66F4;&#x52A0;&#x5BB9;&#x6613;&#x3002;</p><p>&#x897F;&#x95E8;&#x5B50;&#x5468;&#x4E09;&#x8868;&#x793A;&#x4E86;&#x5931;&#x671B;&#x3002;&#x201C;&#x4FDD;&#x62A4;&#x672C;&#x5730;&#x987E;&#x5BA2;&#x5229;&#x76CA;&#x7EDD;&#x4E0D;&#x80FD;&#x610F;&#x5473;&#x7740;&#x6B27;&#x6D32;&#x65E0;&#x6CD5;&#x4E0E;&#x4E2D;&#x56FD;&#x3001;&#x7F8E;&#x56FD;&#x7B49;&#x9886;&#x5148;&#x56FD;&#x5BB6;&#x540C;&#x5904;&#x4E00;&#x4E2A;&#x516C;&#x5E73;&#x7684;&#x7ADE;&#x4E89;&#x73AF;&#x5883;&#xFF0C;&#x201D;&#x8BE5;&#x516C;&#x53F8;&#x9996;&#x5E2D;&#x6267;&#x884C;&#x5B98;&#x51EF;&#x98D2;(Joe Kaeser)&#x8868;&#x793A;&#x3002;</p><p>&#x963F;&#x5C14;&#x65AF;&#x901A;&#x8868;&#x793A;&#xFF0C;&#x8FD9;&#x5BF9;&#x4E8E;&#x6B27;&#x6D32;&#x5DE5;&#x4E1A;&#x662F;&#x4E00;&#x4E2A;&#x201C;&#x660E;&#x663E;&#x7684;&#x632B;&#x6298;&#x201D;&#x3002;&#x963F;&#x5C14;&#x65AF;&#x901A;&#x9996;&#x5E2D;&#x6267;&#x884C;&#x5B98;&#x4EA8;&#x5229;&#x2022;&#x6CE2;&#x5E15;-&#x62C9;&#x6CD5;&#x57FA;(Henri Poupart-Lafarge)&#x8868;&#x793A;&#xFF1A;&#x201C;&#x5982;&#x679C;&#x8BF4;&#x6211;&#x6709;&#x4E00;&#x4E2A;&#x9057;&#x61BE;&#xFF0C;&#x90A3;&#x5C31;&#x662F;&#x6211;&#x7528;&#x4E86;&#x2018;&#x6B27;&#x6D32;&#x51A0;&#x519B;&#x4F01;&#x4E1A;&#x2019;&#x8FD9;&#x4E2A;&#x77ED;&#x8BED;&#x3002;&#x201D;</p><p>&#x7136;&#x800C;&#xFF0C;&#x82F1;&#x56FD;&#x3001;&#x897F;&#x73ED;&#x7259;&#x3001;&#x8377;&#x5170;&#x548C;&#x6BD4;&#x5229;&#x65F6;&#x7684;&#x56FD;&#x5BB6;&#x76D1;&#x7BA1;&#x673A;&#x6784;&#x4E00;&#x81F4;&#x516C;&#x5F00;&#x8868;&#x793A;&#x8FD9;&#x4E24;&#x5BB6;&#x516C;&#x53F8;&#x63D0;&#x8BAE;&#x7684;&#x8865;&#x6551;&#x63AA;&#x65BD;&#x8FDC;&#x8FDC;&#x4E0D;&#x591F;&#xFF0C;&#x5E76;&#x5BF9;&#x8FD9;&#x4E00;&#x5426;&#x51B3;&#x8868;&#x793A;&#x6B22;&#x8FCE;&#x3002;&#x8D1F;&#x8D23;&#x76D1;&#x7BA1;&#x82F1;&#x56FD;&#x94C1;&#x8DEF;&#x7684;&#x82F1;&#x56FD;&#x94C1;&#x8DEF;&#x76D1;&#x7BA1;&#x529E;&#x516C;&#x5BA4;(Office of Rail Regulation)&#x8868;&#x793A;&#xFF1A;&#x201C;&#x6211;&#x4EEC;&#x4ECE;&#x4E00;&#x5F00;&#x59CB;&#x5C31;&#x660E;&#x786E;&#x8868;&#x793A;&#xFF0C;&#x8FD9;&#x6B21;&#xFF08;&#x5408;&#x5E76;&#xFF09;&#x5BF9;&#x82F1;&#x56FD;&#x4E58;&#x5BA2;&#x3001;&#x8D27;&#x8FD0;&#x5BA2;&#x6237;&#x548C;&#x7EB3;&#x7A0E;&#x4EBA;&#x6765;&#x8BF4;&#x90FD;&#x662F;&#x4E00;&#x7B14;&#x7CDF;&#x7CD5;&#x7684;&#x4EA4;&#x6613;&#x3002;&#x201D;</p><p>&#x667A;&#x5E93;&#x5E03;&#x9C81;&#x76D6;&#x5C14;(Bruegel)&#x7684;&#x8BBF;&#x95EE;&#x7814;&#x7A76;&#x5458;&#x9A6C;&#x4FEE;&#x2022;&#x6D77;&#x59C6;(Mathew Heim)&#x8BF4;&#xFF1A;&#x201C;&#x652F;&#x6301;&#x6B27;&#x6D32;&#x51A0;&#x519B;&#x4F01;&#x4E1A;&#x4E0D;&#x662F;&#x6B27;&#x6D32;&#x7ADE;&#x4E89;&#x6CD5;&#x5F8B;&#x7684;&#x804C;&#x80FD;&#x3002;</p><p>&#x201C;&#x76F8;&#x53CD;&#xFF0C;&#x6B27;&#x6D32;&#x9700;&#x8981;&#x4E00;&#x9879;&#x8FDE;&#x8D2F;&#x7684;&#x8BA1;&#x5212;&#xFF0C;&#x4EE5;&#x57F9;&#x80B2;&#x5728;&#x5168;&#x7403;&#x7ADE;&#x4E89;&#x73AF;&#x5883;&#x4E0B;&#x80FD;&#x591F;&#x84EC;&#x52C3;&#x53D1;&#x5C55;&#x3001;&#x800C;&#x65E0;&#x9700;&#x8BC9;&#x8BF8;&#x4FDD;&#x62A4;&#x4E3B;&#x4E49;&#x7684;&#x5173;&#x952E;&#x6B27;&#x6D32;&#x4EA7;&#x4E1A;&#x3002;&#x201D;</p><p>&#x897F;&#x95E8;&#x5B50;-&#x963F;&#x5C14;&#x65AF;&#x901A;&#x63D0;&#x8BAE;&#x5408;&#x5E76;&#x4E4B;&#x9645;&#xFF0C;&#x6B27;&#x6D32;&#x6B63;&#x5904;&#x4E8E;&#x4E00;&#x4E2A;&#x653F;&#x6CBB;&#x8FC7;&#x6E21;&#x671F;&#x3002;&#x4ECA;&#x5E74;5&#x6708;&#x7684;&#x6B27;&#x76DF;&#x9009;&#x4E3E;&#xFF0C;&#x4EE5;&#x53CA;&#x4ECA;&#x5E74;&#x5E03;&#x9C81;&#x585E;&#x5C14;&#x9886;&#x5BFC;&#x5C42;&#x7684;&#x53D8;&#x52A8;&#xFF0C;&#x53EF;&#x80FD;&#x5E26;&#x6765;&#x6539;&#x9769;&#x7ADE;&#x4E89;&#x653F;&#x7B56;&#x7684;&#x673A;&#x4F1A;&#x3002;</p><p>&#x6B27;&#x6D32;&#x53CD;&#x5784;&#x65AD;&#x4E3B;&#x7BA1;&#x627F;&#x8BA4;&#x4E2D;&#x56FD;&#x56FD;&#x6709;&#x4F01;&#x4E1A;&#x6784;&#x6210;&#x7ECF;&#x6D4E;&#x5A01;&#x80C1;&#xFF0C;&#x540C;&#x65F6;&#x4E3A;&#x7ADE;&#x4E89;&#x89C4;&#x5219;&#x505A;&#x4E86;&#x8FA9;&#x62A4;&#x2014;&#x2014;&#x5979;&#x8FA9;&#x79F0;&#xFF0C;&#x8FD9;&#x4E9B;&#x89C4;&#x5219;&#x5E2E;&#x52A9;&#x6B27;&#x76DF;&#x516C;&#x53F8;&#x4FDD;&#x6301;&#x521B;&#x65B0;&#x80FD;&#x529B;&#x548C;&#x56FD;&#x9645;&#x7ADE;&#x4E89;&#x529B;&#x3002;</p><p>&#x7EF4;&#x65AF;&#x7279;&#x683C;&#x6307;&#x51FA;&#xFF0C;&#x6B27;&#x76DF;&#x6709;&#x5176;&#x4ED6;&#x624B;&#x6BB5;&#x6765;&#x5236;&#x5B9A;&#x4EA7;&#x4E1A;&#x653F;&#x7B56;&#xFF0C;&#x5982;&#x5916;&#x6765;&#x6295;&#x8D44;&#x7B5B;&#x67E5;&#x3001;&#x53CD;&#x8865;&#x8D34;&#x4E0E;&#x8D38;&#x6613;&#x9632;&#x5FA1;&#x63AA;&#x65BD;&#xFF0C;&#x4EE5;&#x53CA;&#x63A8;&#x52A8;&#x5C01;&#x95ED;&#x7684;&#x5E02;&#x573A;&#x63D0;&#x4F9B;&#x4E92;&#x60E0;&#x51C6;&#x5165;&#x3002;</p><p>&#x963F;&#x5C14;&#x65AF;&#x901A;&#x7684;&#x6CE2;&#x5E15;-&#x62C9;&#x6CD5;&#x57FA;&#x5468;&#x4E8C;&#x544A;&#x8BC9;&#x300A;&#x8D39;&#x52A0;&#x7F57;&#x62A5;&#x300B;(Le Figaro)&#xFF0C;&#x4E0D;&#x4F1A;&#x7B2C;&#x4E8C;&#x6B21;&#x5C1D;&#x8BD5;&#x5408;&#x5E76;&#xFF0C;&#x4E24;&#x5BB6;&#x516C;&#x53F8;&#x4ECA;&#x540E;&#x5C06;&#x5404;&#x8D70;&#x5404;&#x7684;&#x8DEF;&#x3002;</p><p><i>&#x5E15;&#x7279;&#x91CC;&#x514B;&#x2022;&#x9EA6;&#x5409;(Patrick McGee)&#x6CD5;&#x5170;&#x514B;&#x798F;&#x3001;&#x6234;&#x7EF4;&#x2022;&#x57FA;&#x5965;&#x6069;(David Keohane)&#x5DF4;&#x9ECE;&#x8865;&#x5145;&#x62A5;&#x9053;</i></p><p>&#x8BD1;&#x8005;/&#x548C;&#x98CE;</p></div></div>",
  "author": "罗谢尔•托普兰斯基",
  "date_published": "2019-01-02T06:21:00.000Z",
  "lead_image_url": "http://i.ftimg.net/picture/7/000083507_piclink.jpg",
  "dek": "欧盟反垄断专员没有接受西门子和阿尔斯通提出的合并理由，即这次合并对于欧洲企业与中国国有竞争对手展开竞争是必要的。",
  "next_page_url": null,
  "url": null,
  "domain": null,
  "excerpt": null,
  "word_count": null,
  "direction": null,
  "total_pages": 1,
  "rendered_pages": 1
}

```

Can be reproduced via using [qdaily extractor](https://github.com/postlight/mercury-parser/tree/master/src/extractors/custom/www.qdaily.com) too.

### CODE REVIEW

**Comment by adampash** [comment]
@HenryQW Thanks for the PR! I want to make sure we resolve #263 before proceeding with this. And if you have any thoughts on that bug, I'm all ears.

**Comment by benubois** [comment]
I've been looking into character encoding. I think cheerio encodes all entities when calling `$.html()` by default. The advantage of this is that the output from mercury is less prone to passing on XSS vulnerabilities. The disadvantage is that unrelated characters get converted to entities as well i.e. 

```
德 => &#x5FB7;
```

In order to not encode all entities, the [`cleanAndReturnNode` ](https://github.com/postlight/mercury-parser/blob/b3fa18b6d94e01a3446a8f0f7bd7f46591c2e213/src/extractors/generic/content/extractor.js#L83) function would need to get changed from this:

```javascript
return normalizeSpaces($.html(node));
```

To this:

```javascript
return normalizeSpaces($.html(node, { decodeEntities: false }));
```

In [my use](https://github.com/feedbin/feedbin) of mercury, I sanitize the output from mercury so this would be fine, but I'm guessing mercury wants to not pass on executable javascript?

I saw that [another project](https://github.com/jamsinclair/budou-node/commit/c7fdb1d3e1d803ba0cfbfc663ccea394103b37d9) switched to JSDOM to avoid this. Not sure if that would be a suitable replacement for mercury.

/cc @adampash

**Comment by HenryQW** [comment]
@benubois thanks for the input, this reminds that I did use this method in my other project and it works flawlessly.

https://github.com/HenryQW/api.henry.wang/blob/12d4ac330ca8b796fb1ce0f743579394b779352e/controllers/fullTextController.js#L8-L10

Unfortunately, JSDOM is less performant than cheerio from my past experience.

**Comment by adampash** [comment]
> In [my use](https://github.com/feedbin/feedbin) of mercury, I sanitize the output from mercury so this would be fine, but I'm guessing mercury wants to not pass on executable javascript?

Actually, I think this is probably fine. Mercury sanitizes tags and attributes, and it looks like you're right, this could fix a lot o these problems. 

If someone wants to submit a PR with a test that adds the `{ decodeEntities: false }` bit, I think we can comfortably merge.

File: src/extractors/custom/www.ftchinese.com/index.test.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import assert from 'assert';
+import fs from 'fs';
```
Comment (adampash) [inline src/extractors/custom/www.ftchinese.com/index.test.js]: The web test is failing b/c of this line. If you can move the `fs` import to a require (`const fs = require('fs');`) that should take care of that. It looks like I had forgotten to update the parser generator. Fixing that in another PR.

**Comment by HenryQW** [comment]
@adampash adding `{ decodeEntities: false }` to `cheerio.load` [suggested by you](https://github.com/postlight/mercury-parser/issues/263#issuecomment-462539083) in src/resource/index.js 

https://github.com/postlight/mercury-parser/blob/e033835c7287904371371f922c487e6d0d7d7db8/src/resource/index.js#L63-L79 will skip the decoding of `'`, which results in

```
Helter Shelter: NYC&apos;s Fallout Shelters Basically Don&apos;t Exist Anymore 
```
other than
```
Helter Shelter: NYC's Fallout Shelters Basically Don't Exist Anymore
```

**Comment by adampash** [comment]
> @adampash adding `{ decodeEntities: false }` to `cheerio.load` [suggested by you](https://github.com/postlight/mercury-parser/issues/263#issuecomment-462539083) in src/resource/index.js

I think we want to keep that `cheerio.load` call the same. I believe it's the double-decoding that's caused problems? And that's where @benubois suggested passing the `decodeEntities` option [above](https://github.com/postlight/mercury-parser/pull/264#issuecomment-462913039).

**Comment by HenryQW** [comment]
I did change that but it has no effect on Chinese characters.

**Comment by benubois** [comment]
@adampash I've come to believe that `{decodeEntities: false}` is necessary on both `cheerio.load` and `$.html()`. I think with this combination, cheerio leaves encoding/decoding alone. See where [cheerio firsts encodes](https://github.com/cheeriojs/dom-serializer/blob/d26e453910975f9d980df2d90dd893d89132889d/index.js#L37) and then where it [decodes on output](https://github.com/cheeriojs/dom-serializer/blob/d26e453910975f9d980df2d90dd893d89132889d/index.js#L135)

Here's my [pull request addressing this](#273).

**Comment by HenryQW** [comment]
There is another solution for Chinese, see https://github.com/HenryQW/OpenCC.henry.wang/blob/master/route.js#L7-L21

Not sure how would this merge into mercury-parser thought, probably as a middleware?

**Pull request closed without merging**